packages/mdc-grid-list/index.js

```diff
@@ -28,8 +28,12 @@ export class MDCGridList extends MDCComponent {
   getDefaultFoundation() {
     return new MDCGridListFoundation({
       getOffsetWidth: () => this.root_.offsetWidth,
+      isEmpty: () => {
+        return this.root_.querySelectorAll(MDCGridListFoundation.strings.TILE_SELECTOR).length == 0;
+      },
       getOffsetWidthForTileAtIndex: (index) => {
-        return this.root_.querySelectorAll(MDCGridListFoundation.strings.TILE_SELECTOR)[index].offsetWidth;
+        const tile = this.root_.querySelectorAll(MDCGridListFoundation.strings.TILE_SELECTOR)[index];
+        return tile ? tile.offsetWidth : 0;
```

Comment:
I don't think you need this ternary statement here, since `getOffsetWidthForTileAtIndex` is only called when the number of tiles is greater than 0 (thanks to the short-circuit added at the top of `alignCenter`).